dev/com.ibm.ws.transport.http/src/com/ibm/ws/http/netty/NettyChain.java

```diff
@@ -45,7 +46,7 @@
  */
 public class NettyChain extends HttpChain {
 
-    private static final TraceComponent tc = Tr.register(NettyChain.class, HttpMessages.HTTP_TRACE_NAME, HttpMessages.HTTP_BUNDLE);
+    private static final TraceComponent tc = Tr.register(NettyChain.class, HttpMessages.HTTP_TRACE_NAME, "com.ibm.ws.tcpchannel.internal.resources.TCPChannelMessages");
```

Comment:
This will be adjusted to keep the HTTP_BUNDLE and offset logging responsibility to the Netty framework.